lib/eex/lib/eex/compiler.ex

```diff
@@ -279,17 +279,20 @@ defmodule EEx.Compiler do
   def compile(tokens, opts) do
     file = opts[:file] || "nofile"
     line = opts[:line] || 1
+    source = opts[:source] || "nosource"
```

Comment:
Isn't `:source` always present? :) Maybe we should use `Keyword.fetch!(opts, :source)` instead?